SimPEG/EM/NSEM/SrcNSEM.py

```diff
@@ -24,7 +24,8 @@ class BaseNSEMSrc(FDEMBaseSrc):
     '''
 
     freq = None #: Frequency (float)
-
+    _ePrimary = None  #: Primary electric field
+    _bPrimary = None  #: Primary magnetic field
```

Comment:
@grosenkj: I removed these from the base class (there are scenarios, like a prim-sec where we have derivs on the primary that this is dangerous, so I think it should be handled on a case-by-case basis, rather than making an assumption on the base class)